<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodePrompt</class>
 <widget class="QDialog" name="NodePrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add node</string>
  </property>
  <widget class="QWidget" name="verticalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>250</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Create node</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="weightLabel">
        <property name="text">
         <string>Weight: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="weightDoubleSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QFormLayout" name="selectColorLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="currentColorLabel">
          <property name="text">
           <string>color</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="currentColorPushButton">
          <property name="text">
           <string>Change color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textColorLabel">
        <property name="text">
         <string>Text color: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QFormLayout" name="selectTextColorLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="currentTextColorLabel">
          <property name="text">
           <string>color</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="currentTextColorPushButton">
          <property name="text">
           <string>Change color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
